cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/OperationsTable.js

```diff
@@ -126,6 +126,16 @@ export default class OperationsTable extends Component {
       <div className="grid-body">
         {
           this.props.operations.map((operation, i) => {
+            const descriptionsSplit = operation.description.split('\n');
```

Comment:
Is the description mandatory for all operation? Can it be undefined for any operation? Worth checking if we can prevent js error.